### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by bjosv**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+### 0.11.0 - Sep 15, 2023
+
+* Add event callback for events like 'slotmap updated'.
+* Add connect callback for the sync API.
+* Add redisClusterAsyncConnect2() in API.
```
Comment (zuiderkwast) [inline CHANGELOG.md:5]: Maybe mention what it does? (e.g. "... for fully asynchronous startup" or something.) For other bullets, we mention what the features do but not always the function names.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+* Follow MOVED redirect and update slot mapping concurrently.
+* Update slotmap on error.
+  When connect failed, update slotmap instead of sending command to random node.
+  When command failes (timeout, etc.) schedule slotmap update for next command.
+* Update slotmap when redisClusterCommandToNode() fails.
+* Correct parsing of an IPv6 address in an ASK redirect.
+* Correct handling of XREAD and XREADGROUP.
+* Rename of some types and functions.
+  Old names are still defined for backward compability. HIRCLUSTER_NO_OLD_NAMES
+  can be defined when backward compability is not needed.
```
Comment (zuiderkwast) [inline CHANGELOG.md:16]: Why mention the macro name when we don't even mention the renamed functions and types?

```suggestion
* Rename of some types and functions.
  (Old names are still defined by default for backward compability.)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
+### 0.11.0 - Sep 15, 2023
+
+* Add event callback for events like 'slotmap updated'.
+* Add connect callback for the sync API.
+* Add redisClusterAsyncConnect2() in API.
+* Update the slotmap asynchronously in the async API.
+* Follow MOVED redirect and update slot mapping concurrently.
+* Update slotmap on error.
+  When connect failed, update slotmap instead of sending command to random node.
+  When command failes (timeout, etc.) schedule slotmap update for next command.
```
Comment (zuiderkwast) [inline CHANGELOG.md:10]:
```suggestion
  When command fails (timeout, etc.) schedule slotmap update for next command.
```

**Review by zuiderkwast — COMMENTED** [review]
It's an impressive changelog entry.

Just some comments on the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (bjosv) [reply CHANGELOG.md]: Updated

**Pull request merged**